src/main/java/uk/gov/hmcts/reform/civil/service/robotics/RoboticsNotificationService.java

```diff
@@ -51,4 +55,42 @@ private EmailData prepareEmailData(CaseData caseData) {
             throw new RoboticsDataException(e.getMessage(), e);
         }
     }
+
+    private EmailData prepareEmailDataMultiParty(CaseData caseData) {
+        RoboticsCaseData roboticsCaseData = roboticsDataMapper.toRoboticsCaseData(caseData);
+        String triggerEvent = findLatestEventTriggerReason(roboticsCaseData.getEvents());
+
+        return EmailData.builder()
+            .message(String.format(
+                "Mulitparty claim data JSON is attached for %s",
```

Comment:
I dont think we are attaching the JSON payload for these emails anymore. Im not sure what the content should be of the message - doesnt mention anything specific on the ticket

Reply:
updated message body